extensions/historical-imagery/tests/historical-imagery_item.test.js

```diff
@@ -61,23 +61,4 @@ o.spec('historical-imagery item', () => {
       ),
     ).equals(true)(JSON.stringify(validate.errors));
   });
-
```

Comment:
Why delete this? It's not explained in the commit message.

Reply:
This test is deleted because the eo:bands field is no longer required by the historical imagery extension.

Reply:
I see, the "Changes" stuff in the PR description should ideally be in the commit message for future reference.